apps/mull-ui/src/app/components/chat-bubble-list/chat-bubble-list.scss

```
@@ -0,0 +1,3 @@
+.img {
+  width: 11rem;
```

🧠 Should we use max-width so that we don't stretch smaller images but still shrink bigger images?
```suggestion
  max-width: 11rem;
```